src/MgvHasOffers.sol

```diff
@@ -67,6 +67,7 @@ contract MgvHasOffers is MgvCommon {
     uint tickScale,
     MgvStructs.OfferPacked offer,
     MgvStructs.LocalPacked local,
+    Tick bestTick,
```

Comment:
Is it more efficient/less gas costly to take this as a separate parameter than to do `bestTick = local.bestTick()` in the beginning of function?

In all calls to `dislodgeOffer` we have `bestTick == local.bestTick()`, so this parameter isn't strictly needed (but may save gas?).

Reply:
It's already computed before one of the calls (in `writeOffer`) so it prevents re-computing it. I haven't measured though